core/src/main/scala/kafka/server/ControllerServer.scala

```diff
@@ -158,6 +159,8 @@ class ControllerServer(
 
       createTopicPolicy = Option(config.
         getConfiguredInstance(CreateTopicPolicyClassNameProp, classOf[CreateTopicPolicy]))
+      topicPolicy = Option(config.
```

Comment:
nit: it's probably best to keep the old configs (`createTopicPolicy` and `alterConfigPolicy`) together and not insert this new one in the middle

Reply:
Ok.